aspnetcore/migration/22-to-30.md

```diff
@@ -69,3 +69,22 @@ Newtonsoft settings can be set with `AddNewtonsoftJson`:
                options.SerializerSettings.ContractResolver = 
                   new CamelCasePropertyNamesContractResolver());
     ```
+
+## HostBuilder replaces WebHostBuilder
+
+The ASP.NET Core 3.0 templates use [Generic Host](xref:fundamentals/host/generic-host). Previous versions used [Web Host](xref:fundamentals/host/web-host). The following code shows the ASP.NET Core 3.0 template generated `Program` class:
+
+[!code-csharp[](22-to-30/samples/Program.cs?name=snippet)]
+
+The following code shows the ASP.NET Core 2.2 template-generated `Program` class:
+
+[!code-csharp[](22-to-30/samples/Program2.2.cs?name=snippet)]
+
+<xref:Microsoft.AspNetCore.Hosting.IWebHostBuilder> remains in 3.0 and is the type of the `webBuilder` seen in the preceding code sample. <xref:Microsoft.AspNetCore.Hosting.WebHostBuilder> will be deprecated in a future release and replaced by `HostBuilder`.
+
+## Moving from WebHostBuilder to HostBuilder
+
+The most significant change from `WebHostBuilder` to `HostBuilder` is in [dependency injection (DI)](xref:fundamentals/dependency-injection). `HostBuilder` can only inject <xref:Microsoft.Extensions.Configuration.IConfiguration> and <xref:Microsoft.Extensions.Hosting.IHostingEnvironment>. The `HostBuilder` DI limitations:
```

Comment:
```suggestion
The most significant change from `WebHostBuilder` to `HostBuilder` is in [dependency injection (DI)](xref:fundamentals/dependency-injection). When using `HostBuilder` you can only inject <xref:Microsoft.Extensions.Configuration.IConfiguration>, <xref:Microsoft.Extensions.Hosting.IHostingEnvironment>, and <xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment> into Startup's constructor. The `HostBuilder` DI constraints:
```

Reply:
Comma  ....

> When using \`HostBuilder\`, you can ...

maybe "**the** Startup's constructor" ?